en/includes/references/app-native-authentication.md

```diff
@@ -471,7 +471,11 @@ The application goes through the following steps to complete app-native authenti
 
 ### Scenario 3: User selects passkey login out of multiple options
 
-If a login step has multiple login options, the application goes through the following steps to complete app-native authentication.
+If a login step has multiple login options, the application goes through the following steps to complete passkey login with app-native authentication.
+
+!!! note "Prerequisite"
+
+    If you wish to integrate passkeys with app-native authentication for a mobile application, it is mandatory to make it a [FIDO trusted app]({{base_path}}/guides/authentication/passwordless-login/add-passwordless-login-with-passkey/#make-application-a-fido-trusted-app).
```

Comment:
Instead of `it is mandatory to make it a FIDO trusted app`, let's reword this into something like `it is mandatory to establish a secure association between {{product_name}} and your application. Refer FIDO trusted app section.`